### ISSUE TITLE
pip installation

### ISSUE THREAD

**Issue by sjanssen2** [linnabrown/run_dbcan#43]
You suggest to use conda environments for installation, however your recommendation to install dbcan itself `pip install run-dbcan==2.0.9 --user` would currently **not** install the package into the newly created environment, but in a general user directory. May I suggest to remove the `--user` flag if the user decides to install within conda?

**Comment by linnabrown** [comment]
Good suggestion, I will remove this flag. Thank you.

**Comment by gcagle1** [comment]
Hello, I am installing on my university HPC and when I do 
```pip install run-dbcan==2.0.11```
I got a permission error:
PermissionError: [Errno 13] Permission denied: '/usr/local/packages/python/3.6.4-anaconda/lib/python3.6/site-packages/natsort-7.1.1.dist-info'

However I saw this comment thread and gave it a try with the ```--user``` flag. When I ran 
```pip install run-dbcan==2.0.11 --user```
I was able to install successfully without permission error.

I just wanted to let you know (and others who maybe have this issue).

Thank you very much for the software and helpful documentation.

Best wishes,
Grace
